stdpopsim/genomes.py

```diff
@@ -390,15 +447,27 @@ def species_contig(
             if mutation_rate is None:
                 mutation_rate = chrom.mutation_rate
 
+            if species.genome.bacterial_recombination is True:
+                gene_conversion_length = chrom.gene_conversion_length
+
             if use_species_gene_conversion is True:
-                gene_conversion_rate = chrom.gene_conversion_rate
+                gene_conversion_fraction = chrom.gene_conversion_fraction
                 gene_conversion_length = chrom.gene_conversion_length
+                if (
+                    gene_conversion_fraction is not None
+                    and gene_conversion_fraction > 0
+                ):
+                    recomb_map = msprime.RateMap(
+                        position=recomb_map.position,
+                        rate=recomb_map.rate / (1 - gene_conversion_fraction),
+                    )
```

Comment:
this patterns comes up a number of times in the code now. I wonder if it's worth writing a couple utility funcs that go from recomb_map to gc adjusted recomb_map?

Reply:
I've had a look, and I don't think we actually do the same thing twice?